app/src/main/java/org/mozilla/fenix/addons/NotYetSupportedAddonFragment.kt

```diff
@@ -26,11 +28,27 @@ private const val LEARN_MORE_URL =
  * Fragment for displaying and managing add-ons that are not yet supported by the browser.
  */
 class NotYetSupportedAddonFragment :
-    Fragment(R.layout.fragment_not_yet_supported_addons), UnsupportedAddonsAdapterDelegate {
+    Fragment(), UnsupportedAddonsAdapterDelegate {
 
     private val args by navArgs<NotYetSupportedAddonFragmentArgs>()
     private var unsupportedAddonsAdapter: UnsupportedAddonsAdapter? = null
 
+    private var _binding: FragmentNotYetSupportedAddonsBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
```

Comment:
Like in previous cases, maybe we can keep the Fragment constructor which takes the layout parameter, bind and use the layout in place in `onCreateView` and in so avoid all this ceremony?